test/testDat/testModules/ancient/Schiffels_2016/POSEIDON.yml

```diff
@@ -1,4 +1,4 @@
-poseidonVersion: 2.4.0
```

Comment:
Wow, I just realised we still call this directory `testModules`, even though we've adopted the name "Package" for a long time now... not so important, of course, but perhaps worth a quick commit.

Reply:
Right! That's a ToDo.

Reply:
Done.